README.md

```diff
@@ -80,6 +80,50 @@ case object GetPeople extends CollectionQuery[Seq, Person] {
 val person = db(GetPeople).head // Person(Coda Hale,chale@example.com,29)
 ```
 
+You can also use the sql string interpolator:
```

Comment:
Yes. You're right.

The style is definitely different, but IMO my superior. More concise. Less error prone. Easier to read.

With that said, there is currently a major drawback to the interpolator. Since there are no longer classes per query, we don't get metrics on each distinct query for free.

Internally at Simple we have an instrumentation library that allows us to place an `@Instrumented` macro annotation on any method and get metrics for free. We use that for interpolated queries. I think we need to OSS that library to provide a recommendation for equivalent functionality on interpolated SQL.

We need to come on some consensus how we should proceed. I'm thinking...